pkg/kademlia/peer_discovery.go

```diff
@@ -147,57 +151,37 @@ type queueItem struct {
 }
 
 // newDiscoveryQueue returns a items with priority based on XOR from targetBytes
-func newDiscoveryQueue(size int) *discoveryQueue {
+func newDiscoveryQueue(target storj.NodeID, K int) *discoveryQueue {
```

Comment:
why is K uppercase here? in the context of a queue `size` makes more sense than `k`

Reply:
addressed in https://github.com/storj/storj/pull/2252